### PR TITLE
Init upgrade command

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/KAC-37

### CODE REVIEW

File: internal/pkg/cli/dialog/delete_template.go [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-type deleteTmplDialog struct {
-	*Dialogs
-	projectState *project.State
-	options      *options.Options
-	out          deleteTemplate.Options
-}
-
-// AskDeleteTemplateOptions - dialog for deleting a template from the project.
-func (p *Dialogs) AskDeleteTemplateOptions(projectState *project.State, opts *options.Options) (deleteTemplate.Options, error) {
-	dialog := &deleteTmplDialog{
```
Comment (webrouse) [inline internal/pkg/cli/dialog/delete_template.go:18]: Presunute vid `template_instance.go`.

File: internal/pkg/cli/dialog/upgrade_template.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/options"
+	"github.com/keboola/keboola-as-code/internal/pkg/project"
+	"github.com/keboola/keboola-as-code/internal/pkg/template"
+	upgradeTemplate "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/template/upgrade"
+)
+
+type upgradeTmplDialog struct {
+	*Dialogs
+	projectState *project.State
```
Comment (webrouse) [inline internal/pkg/cli/dialog/upgrade_template.go:12]: Upgrade dialog, tiez iba initnuty, bez testov a nastavovania default hodnot.

File: internal/pkg/cli/dialog/use_template.go [hunk 3]
```diff
@@ -105,8 +90,42 @@ func (d *useTmplDialog) askInstanceName() (string, error) {
 	return v, nil
 }
 
-func (d *useTmplDialog) askInputs(stepsGroups input.StepsGroupsExt) error {
-	return stepsGroups.VisitInputs(func(group *input.StepsGroupExt, step *input.StepExt, inputDef *input.Input) error {
+type useTmplInputsDialog struct {
```
Comment (webrouse) [inline internal/pkg/cli/dialog/use_template.go:93]: z `useTmplDialog` som oddelil `useTmplInputsDialog`, ktory pouzijem aj v `upgrade` commande.

**Review by webrouse — COMMENTED** [review]
V tomto PR som initol command `local template upgrade` + nachystal som si CLI dialog a operaciu, v dalsom PR bude implementacia.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**